### DISCUSSION TITLE
Where has the music note gone?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5080] [General]
Need to add music notes to these lines but it's missing from the right click tools

![image](https://user-images.githubusercontent.com/82276985/120359834-208f8a80-c2d6-11eb-9859-59d858ca18ef.png)

**Comment by DrReddP** [comment]
Um ok so I put in # instead, then used "fix common errors" to switch them to music notes, but when I hit save, this pops up and when I reopen the file they're # again. I tried both the regular SE and the beta

![image](https://user-images.githubusercontent.com/82276985/120360860-3a7d9d00-c2d7-11eb-9369-b8294d244e89.png)

**Reply by OmrSi** [reply]
That's because you have the wrong encoding, this encoding doesn't support the music symbol, you should switch to UTF-8.

**Comment by DrReddP** [comment]
Where do I change the encoding? I checked several other srt's on the same
computer using the same SE and they all save music notes just fine

**Reply by OmrSi** [reply]
![image](https://user-images.githubusercontent.com/20923700/120366902-e1365d80-c318-11eb-9c8d-acdacc64b860.png)

**Reply by OmrSi** [reply]
I bet you'll find it's UTF-8 in the other files.

**Comment by DrReddP** [comment]
Perfect, thank you

**Reply by OmrSi** [reply]
You got it. :D
